BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/Dtos/HearingDto.cs

```diff
@@ -14,5 +14,6 @@ public class HearingDto
         public bool RecordAudio { get; set; }
         public Guid? GroupId { get; set; }
         public string HearingType { get; set; }
+        public string JudgeDisplayName { get; set;}
```

Comment:
As mentioned in the Bookings API PR, this property does not belong in the hearings dto

Reply:
I am not sure what you mean. Can you please suggest solution for it?

Reply:
added new dto